crates/std_detect/src/detect/arch/loongarch.rs

```diff
@@ -16,20 +16,20 @@ features! {
     /// * `"lbt"`
     /// * `"lvz"`
     /// * `"ual"`
-    #[unstable(feature = "stdarch_loongarch_feature_detection", issue = "117425")]
-    @FEATURE: #[unstable(feature = "stdarch_loongarch_feature_detection", issue = "117425")] f: "f";
+    #[stable(feature = "stdarch_loongarch_feature", since = "1.88.0")]
+    @FEATURE: #[stable(feature = "stdarch_loongarch_feature", since = "1.88.0")] f: "f";
```

Comment:
Should I replace `1.88.0` with the `CURRENT_RUSTC_VERSION`?

Reply:
CURRENT_RUSTC_VERSION doesn't work in submodules like stdarch, you have to put the actual version.

Reply:
Okay. Thanks!